src/main/java/com/blackduck/integration/rest/RestConstants.java

```diff
@@ -14,7 +14,7 @@
 import java.util.TimeZone;
 
 public class RestConstants {
-    public static final String JSON_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSX";
+    public static final String JSON_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
```

Comment:
Will this work against older hubs, say ones before 2025.4.1? Detect tends to be compatible with a few releases of hub so if we pulled this in would we have problem with earlier releases?

Reply:
@dterrybd 
Because of this question I did the same test as outlined in the description, except using 2025.1.1 of BD, which is before the change to Joda-Time. Alert was able to connect and use the endpoints it uses which contain DateTime fields in them without issue.

While I think this test indicates that Detect should not have a problem using the change in this MR against older BD deploys, I can't definitively say Detect won't have any issues. Regression testing will need to be run to prove this out for Detect.